catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -145,13 +157,13 @@ public DynamicSchemaResolver() {
 
     /**
      * Adds the fields that are already in the server to the cache. This method should be called
-     * once the SolrServer is up to ensure the cache is synchronized with the server.
+     * once the SolrClient is up to ensure the cache is synchronized with the server.
      *
-     * @param server
-     *            the SolrServer we are working with
+     * @param client
+     *            the SolrClient we are working with
      */
-    public void addFieldsFromServer(SolrServer server) {
-        if (server == null) {
+    public void addFieldsFromServer(SolrClient client) {
+        if (client == null) {
             LOGGER.warn("Server is null, could not add fields to cache.");
```

Comment:
There are many references to server in method names and logs that I skipped but now I think I should fix all of them to prepare for the switch to Solr cloud.